pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -68,6 +68,7 @@ let
           ++ lib.optional (cfg.enableGnomeExtensions or false) gnome-browser-connector
           ++ lib.optional (cfg.enableUgetIntegrator or false) uget-integrator
           ++ lib.optional (cfg.enablePlasmaBrowserIntegration or false) plasma5Packages.plasma-browser-integration
+          ++ lib.optional (cfg.enableFirefoxPwa or false) firefoxpwa.unwrapped
```

Comment:
Hi, I tried your PR and it works great! After a few tweaks though: the Firefox addon was stuck at the "download runtime" step, so I had to ```cp -Lr ${pkgs.firefox}/lib/firefox ~/.local/share/firefoxpwa/runtime```. Then I also have some write permissions issues with ```./_autoconfig.cfg``` and ```./defaults/pref/autoconfig.js``` at each launch of a PWA. So do you think there would be a fix for this in order to maybe not have to manually copy paths from Nix Store? Thanks!

Reply:
I did not manage to reproduce it. Have you tried to remove all the shortcuts and the firefox profiles created by firefoxpwa, and also `~/.local/share/firefoxpwa` before reinstalling both the package and the firefox extension?

Reply:
I'm going to try that. Also it might help to mention that I am running Firefox 110.0.1, is it the same as you? Also, it seems that I am not the only one with a similar issue, so it might not be related to the way it is packaged in NixOS: https://github.com/filips123/PWAsForFirefox/issues/309.

Reply:
Ok, I removed everything and reinstalled (now the runtime installs automatically) and it seemed that it comes from a conflict between 2 profiles about those two config files I mentioned. Can you reproduce those two files being read-only after having launched a PWA for the first time? Thank you!

Reply:
yep it seem that sometimes those files are oput read only for some reason.

Reply:
I think this is a upstream problem

Reply:
This can also be added to https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/programs/firefox.nix.